test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -472,5 +472,26 @@ public void AddProductWithDifferentPropertiesTest(SupportedLanguages lang)
             // Verify result
             Assert.Equal("test2", this.ExecuteScalar($"select Name from Products where ProductId=0"));
         }
+
+        /// <summary>
+        /// Tests that an error is thrown when the upserted item contains a unsupported column type.
+        /// </summary>
+        [Theory]
+        [SqlInlineData()]
+        [UnsupportedLanguages(SupportedLanguages.OutOfProc)]
+        public async Task AddProductUnsupportedTypesTest(SupportedLanguages lang)
+        {
+            var foundExpectedMessageSource = new TaskCompletionSource<bool>();
+            this.StartFunctionHost(nameof(AddProductUnsupportedTypes), lang, true, (object sender, DataReceivedEventArgs e) =>
+            {
+                if (e.Data.Contains("The type(s) of the following column(s) are not supported: Text, Ntext, Image."))
```

Comment:
nit: I was initially confused since it seemed to say the type instead of the column names. So maybe name the columns something like `TextCol` just to make it clear